nix/vast/plugins.nix

```diff
@@ -0,0 +1,7 @@
+{
+  example = {
+    src = ../../examples/plugins/example;
+    buildInputs = [];
+    cmakeFlags = [];
+  };
+}
```

Comment:
It looks like this needs to be written for every plugin. That's a non-starter—we cannot require plugins to be available in the build tree, because we want to support a static build including closed-source plugins.